File: 1-js/1-js-basics/3.1-conditional-statements-advanced-lab/07-workingHours.js
function solve([time, day]) {

    time = Number(time);

    let workingDays = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];

    return workingDays.includes(day) && time >= 10 && time <= 18 ? 'open' : 'closed';

}

console.log(solve(['11', 'Monday'])); // open

console.log(solve(['19', 'Friday'])); // closed

File: 2-python/1-python-basics/3.1-conditional-statements-advanced-lab/07-working_hours.py
hour = int(input())
day = input()
status = ''

if day == 'Monday' \
        or day == 'Tuesday' \
        or day == 'Wednesday' \
        or day == 'Thursday' \
        or day == 'Friday' \
        or day == 'Saturday':
    if 10 <= hour <= 18:
        status = 'open'
    else:
        status = 'closed'
else:
    status = 'closed'

print(status)
